debian/gbp.conf

```diff
@@ -19,6 +19,8 @@ prebuild = bash -c "
     # Create upstream tarball from reference, exclude items that do not belong
     pushd $GBP_GIT_DIR/.. &&
     ls -1 &&
+    # TODO: remove this line and the next after the first time the change is pushed
+    echo '/.* !export-ignore' >$GBP_GIT_DIR/info/attributes &&
```

Comment:
I assume we cannot do this as a separate commit in this PR since Evergreen would consider all commits part of one patch.

Reply:
True.  The alternative is to remove this and then live with the first Evergreen waterfall build containing this change failing.  All subsequent changes should then work as expected.  I am happy to go either with it.

Reply:
I'm fine with doing a follow up.